vm/parser.ts

```diff
@@ -759,8 +764,17 @@ module J2ME {
       var s = this;
       var methods_count = s.readU2();
       var m = this.methods = new Array(methods_count);
+      var ids = this.methodIDs = new Array(methods_count);
       for (var i = 0; i < methods_count; i++) {
         m[i] = s.offset;
+
+        ids[i] = [
```

Comment:
> We should try to avoid this, because it reduces the effectiveness of lazily loading classes

As long as we need that info to minimally link the methods, it doesn't actually reduce the effectiveness of lazily loading the classes, because we'll load the info at link time anyway.

> and also uses more memory (because we're storing the method name and signature both here and in MethodInfo)

We can fix that by deleting the structure after minimally linking the methods.